tests/test_features.py

```diff
@@ -395,3 +396,56 @@ class Truthy(betterproto.Message):
     assert t
     t.bar = 0
     assert not t
+
+
+def test_iso_datetime():
+    @dataclass
+    class Envelop(betterproto.Message):
+        ts: datetime = betterproto.message_field(1)
+
+    e = Envelop()
+    # valid ISO datetimes according to https://www.myintervals.com/blog/2009/05/20/iso-8601-date-validation-that-doesnt-suck/
+    candidates = """2009-12-12T12:34
```

Comment:
This set of test cases looks convincing but I'm curious what case you encountered that didn't behave as expected before?

Reply:
I didn't check all of them. I got to it because it didn't work with nanoseconds ("2010-02-18T16:00:00.23334444")